pipelines/_ingredients/ApPipeCalibrateImage.yaml

```diff
@@ -0,0 +1,90 @@
+description: End to end Alert Production pipeline.
+# Look in subdirectories of $AP_PIPE_DIR/pipelines to find customized pipelines
+# for each camera. Those pipelines import this general AP pipeline.
+#
+# NOTES
+# Remember to run make_apdb.py and use the same configs for diaPipe
+# A db_url is always required, e.g.,
+# -c diaPipe:apdb.db_url: 'sqlite:////project/user/association.db'
+# Option to specify connection_timeout for sqlite APDBs encountering lock errors, i.e.,
+# -c diaPipe:apdb.connection_timeout: 240
+
+# WARNING: camera-specific pipelines importing this pipeline may
+# blow away all the configs that are set in this file.
+# To update a pipeline config prior to DM-35504, you MUST put it in either,
+# e.g., $AP_PIPE_DIR/config/$CAMERA/someTask.py, or in a camera-specific,
+# pipeline, e.g., $AP_PIPE_DIR/pipelines/$CAMERA/ApPipe.yaml.
+
+imports:
+  - location: $AP_PIPE_DIR/pipelines/_ingredients/ProcessCcdCalibrateImage.yaml
+parameters:
+  # Pipeline configurable to run on both goodSeeing and deep templates, depending on dataset.
+  coaddName: goodSeeing
+tasks:
+  retrieveTemplate:  # For multi-tract difference imaging
+    class: lsst.ip.diffim.getTemplate.GetTemplateTask
+    config:
+      connections.bbox: initial_pvi.bbox
+      connections.wcs: initial_pvi.wcs
```

Comment:
I suppose it's outside the scope of this issue, but why does `calibrateImage` have outputs that are incompatible with the old `calibrate`? That will force _all_ pipelines (both AP and DRP) to have lengthy overrides of this sort.

Reply:
Looking at the `ap_verify` changes, I think it might be less disruptive to rename the `calibrateImage` outputs to the `calibrate` convention. As it is, a _lot_ of metrics need to be reconfigured.

Reply:
That was a deliberate choice following the design laid out in [DM-37584](https://jira.lsstcorp.org/browse/DM-37584). These are the names we want for these products eventually anyway, drp will mostly be making new tasks themselves per DMTN-172, and the outputs are different, though currently subtly: `initial_pvi` is photometrically calibrated while `calexp` is not, and `initial_stars_footprints_detector` is a much reduced list of sources compared with `src` (and will eventually only contain the footprints, and instead we'll have a parquet table for the sources themselves).